### PR TITLE
Address cases of multiple gene associations

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/8329

Please see ticket for explanation of changes

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -271426,6 +271422,7 @@ is_a: MONDO:1040055 {source="https://clinicalgenome.org/working-groups/clinical-
 intersection_of: MONDO:0019200 ! retinitis pigmentosa
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 ! ROM1
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9942 ! PRPH2
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:271425]: I don't think this is correct. 
According to [OMIM](https://www.omim.org/entry/608133?search=608133&highlight=608133), this record refers to the disease for which the RDS gene (PRPH2) is mutated, however, "A digenic form of retinitis pigmentosa resulting from a mutation in the RDS gene ([179605.0004](https://www.omim.org/entry/179605#0004)) and a null mutation of the ROM1 gene (see [180721.0001](https://www.omim.org/entry/180721#0001)) has been reported."

Therefore, for this record, we cannot say that in 100% of the case, the disease is caused by a mutation in both genes (or in a mutation in only one gene). 

The gene annotations should be removed for this disease. 

Note: the definition for this disease should also be changed to make this disease concept not specific to the mutation in the PRPH2 gene.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
I think retinitis pigmentosa 7 should not have association with any genes. Please see my explanation in the comment. I can make the changes if you want, but I first want to make sure you have a chance you check my explanation in case you don't disagree.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:271425]: Thanks Sabrina. I agree and will make the changes.

Comment (twhetzel) [reply src/ontology/mondo-edit.obo]: Updated

**Review by sabrinatoro — APPROVED** [review]
@twhetzel I updated the definition for retinitis pigmentosa 7.
I think this PR can be merged after QC have passed. 

[OMIM:608133](https://omim.org/entry/608133) indicates that there are 2 included entries. We should add these with the "omim included" x-ref in a different PR (note, i have not created an issue for this).

**Pull request merged**